src/js/scroller.js

```diff
@@ -15,11 +15,62 @@ RiseVision.Common.Scroller = function (prefs) {
     _items = [],
     _xpos = 0,
     _originalXpos = 0,
+    _customFonts = [],
     _utils = RiseVision.Common.Utilities;
 
   /*
    *  Private Methods
    */
+
+  function getCustomFonts() {
+    for (var i = 0; i < _items.length; i++) {
+      if (_items[i].fontStyle.font.type && (_items[i].fontStyle.font.type === "custom")) {
+        _customFonts.push({
+          "family": _items[i].fontStyle.font.family,
+          "url": _items[i].fontStyle.font.url
+        });
+      }
+    }
+  }
+
+  function loadCustomFonts() {
```

Comment:
@donnapep I'm wondering if we should look to replace our existing function in `common.js` here https://goo.gl/zj0dFd with this one that uses WebFontLoader since we're starting to use WebFontLoader in all our widgets. If we did you could leverage `loadCustomFonts` there instead of repeating here. What you think?